Lux/README.md

```diff
@@ -21,6 +21,7 @@ For an overview of the development history of the project see the [Jen README](h
 Via Emscripten, C++ and JavaScript can have access to shared memory space. This allows the ImagePortCanvas to be rendered in 1-2 milliseconds once the image is rendered.
 
 ## API
+<br><img src="https://github.com/joyhughes/joyhughes.github.io/blob/main/request_response.png" height = 400><br>
```

Comment:
recommend dropping github.com part of URI so you can move this website off github if you ever want to more easily

Reply:
i think it's just `/joyhughes/joyhughes.github.io/blob/...`

Reply:
Didn't see this before I merged. Where do you recommend putting images going forward?

Reply:
The image link you mention is on github.com. I can tell by reading the URI. So you don't need that hostname as part of that URI I believe nor do you need the https part.

Reply:
Got your response on Discord to use relative URI